server/routes/prisoner/handlers/overview.ts

```diff
@@ -16,10 +16,6 @@ export default class OverviewRoutes {
     const { prisoner } = req
     const { token, activeCaseLoadId, username } = res.locals.user
     const bookingId = prisoner.bookingId as unknown as number
-    const activeCourtCaseCount = await this.prisonerService.getActiveCourtCaseCount(bookingId, token)
```

Comment:
this check is no longer required.. it is handled elsewhere in a less restrictive manner